docs/user-guides/tools.md

```diff
@@ -1,7 +1,51 @@
 # Tools
 
-Every tool should be explained here
+There are additional tools in Flowchem to help users create the configuration file and utilize the API server.
 
-`flowchem-autodiscover` is a tool that helps to prepare the configuration files.
+## Autodiscover
 
-`get_all_flowchem_devices`
\ No newline at end of file
+Some devices implemented in Flowchem can be discovered using the autodiscover function present in Flowchem. To activate
+this function, simply type the command in the command window.
+
+```shell
+flowchem-autodiscover
+```
+
+Autodiscover will examine the Ethernet input by sending and analyzing the data to identify the device connected to the
```

Comment:
i think it examines the local network by sending out a broadcast, check and correct. Also, it will iterate through serial prots if asked to, change wording

Reply:
Done!